docker-compose.yml

```diff
@@ -16,24 +16,68 @@ services:
     image: postgis/postgis:13-3.2-alpine
     volumes:
       - pg_data:/var/lib/postgresql/data
+      # copy the sql script to create db/tables
+      - ./sql/init_db.sql:/docker-entrypoint-initdb.d/init_db.sql
     environment:
-      - POSTGRES_PASSWORD=postgres
-      - POSTGRES_USER=postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_USER: postgres
+    restart: always
     ports:
       - 5432:5432
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      start_period: 60s
+      interval: 5s
+      timeout: 3s
+      retries: 10
 
   api_service:
     container_name: api_service
     build:
         context: .
-        dockerfile: Dockerfile
+        dockerfile: Dockerfile.ecs
     image: wcs/mermaid-api:latest
-    volumes:
-      - $PWD/src:/var/projects/webapp
     ports:
-      - 8080:8080
+      - 8081:8081
+    environment:
+      # db
+      DB_NAME: mermaid
+      DB_HOST: api_db
```

Comment:
Yeah, I don't like how we are specifying the `container_name` in the compose files either... I we remove them, then the container names will be scoped to their directory name, ie: `mermaid-api_db`. Alternatively, you can specify PROJECT_NAME and docker will prefix your containers to that, instead of the folder name.